imageio/plugins/ffmpeg.py

```diff
@@ -626,6 +636,32 @@ def _initialize(self):
                     quality = int(quality*30)+1
                     cmd += ['-qscale:v', str(quality)]  # for others
             cmd += ['-r', "%d" % fps]
+
+            # Note, for most codecs, the image dimensions must be divisible by
+            # 16 the default for the macro_block_size is 16. Check if image is
+            # divisible, if not have ffmpeg upsize to nearest size and warn
+            # user they should correct input image if this is not desired.
+            macro_block_size = self.request.kwargs.get('macro_block_size', 16)
+            if macro_block_size > 1 and \
+                    (self._size[1] % macro_block_size > 0 or
+                        self._size[0] % macro_block_size > 0):
+                out_w = self._size[1] + macro_block_size - \
+                    (self._size[1] % macro_block_size)
+                out_h = self._size[0] + macro_block_size - \
+                    (self._size[0] % macro_block_size)
+                cmd += ['-vf', 'scale={0}:{1}'.format(out_w, out_h)]
+                print("IMAGEIO FFMPEG_WRITER WARNING: input image is not "
```

Comment:
Ok, I'll change it to warnings.warn. I like logging better but if no logger is setup by the user they won't see the warning.